datamade/just-spaces
# Two track surveys

## Overview

This is the last bit of work needed to close #46! https://github.com/datamade/django-fobi/pull/2 will need to be merged before it passes checks. 

### To do 

- [x] Write a test or two!

## Testing Instructions

 * If you don't already have it, create the database: `pg_restore -C -j4 --no-owner just-spaces.dump | psql`
* Migrate: `python manage.py migrate`
* Login as testuser (password in the DataMade LastPass under "Just Spaces Staging")
* Create a new survey with type Observational
* When you go to add a new question, do you see the "Observational" category?
* Create a new survey with type Intercept
* When you go to add a new question, do you NOT see the "Observational" category?

Connects #46
